### DISCUSSION TITLE
Error: "Request failed with status 500 due to session not created: This version of ChromeDriver only supports Chrome version 98 Current browser version is 101.0.4951.4

### DISCUSSION THREAD

**Discussion by ellatom** [webdriverio/webdriverio#8285] [Technical Discussion]
Hi,
I'm getting error when running automation on electron app with version lower than google chrome installed version.
I can't downgrade the google chrome version.
Is there a way to run automation on computer with the following:
google chrome version 101.0.4951.41 
electron app version 100.0.4896.60
chrome driver version 101.0.4951.41  or 100.0.4896.60
Thanks,Ella

**Comment by christian-bromann** [comment]
> I'm getting error when running automation on electron app

Have you tried the [`wdio-electron-service`](https://www.npmjs.com/package/wdio-electron-service) for this? It is build to take away some of these setup issues.

**Reply by ellatom** [reply]
@christian-bromann I'm using wdio-electron-service. 
configuration fle 'wdio.conf.ts':
```
  services: [
    [
      'electron',
      {
        newSessionPerTest: false,
        binaryPath: process.env.RELATIVE_PATH_TO_ELECTRON, 
        appArgs: ['--disable-infobars', '--disable-dev-shm-usage', '--no-sandbox', '--remote-debugging-port=9222'],
        chromedriver: {
          port: 9519,
          logFileName: 'wdio-chromedriver.log',
        },
        safaridriver: {
          port: 4444,
          logFileName: 'wdio-safaridriver.log',
        },
      },
    ],
```

**Reply by christian-bromann** [reply]
Please raise an issue in that project then, thank you!

**Reply by ellatom** [reply]
Thanks @christian-bromann  I have opened an issue

**Comment by amer2019IIUM** [comment]
Hi @ellatom, I had the same issue. When I run `npm i -D chromedriver@100`, it worked fine. Try to update the chromedriver (https://www.npmjs.com/package/wdio-electron-service)

**Reply by Jaimin180296** [reply]
@amer2019IIUM  your solution worked able to execute the test I installed the chrome version as per the error logs it worked. 

npm i -D chromedriver@104

**Reply by paulina-piotrowska** [reply]
Worked, thanks so much!

**Comment by teodormaxim99** [comment]
I encountered this issue while running ''npm run test''. Running ''npm run headless'' worked perfectly.
Go to the project's folder and then node_modules\selenium-standalone\.selenium and delete chromedriver folder from there. When running again it should download the necessary version and the browser window should open.
